SplunkRumCrashReporting/SplunkRumCrashReporting/CrashReporting.swift

```diff
@@ -24,6 +24,9 @@ let CrashReportingVersionString = "0.2.0"
 
 var TheCrashReporter: PLCrashReporter?
 
+let keyID   = "SessionID"
```

Comment:
Might rename it `sessionIdKey` and `screenNameKey` or something similar, at the moment it's a bit hard to understand at first glance what is `keyID` and `keyName`